src/set_theory/surreal/basic.lean

```diff
@@ -284,9 +294,28 @@ noncomputable instance : linear_ordered_add_comm_group surreal :=
   decidable_le := classical.dec_rel _,
   ..surreal.ordered_add_comm_group }
 
+end surreal
+
+open surreal
+
+namespace ordinal
+
+/-- Converts an ordinal into the corresponding surreal. -/
+noncomputable def to_surreal (o : ordinal) : surreal :=
```

Comment:
Are any of the following trivial?

* `0.to_surreal = 0`
* `1.to_surreal = 1`
* `(n : ordinal).to_surreal = n` for `n : nat`

If not, fine to leave to a later PR.

Reply:
Oddly enough, we're currently missing the `to_pgame` analogs. `n.to_pgame = n` is nontrivial and it's probably best to wait for natural addition to prove that. The other two lemmas could be proven easily.

Reply:
Yeah, I figured the `nat.cast` case would be tricky. Is the `0` case rfl?

Reply:
Nope. `to_pgame 0` isn't def-eq to `0` either.